### PR TITLE
Adds bot oxygenLevel and breath level change emitter.

### LINKED ISSUE #1882: Oxygen level(underwater)
I'd like to have event on change of "oxygen level"(these bubbles over hunger in regular mc client), I don't even know how minecraft names this bar. A property like bot.breathLevel would be useful too(e.g. to prevent bot from dying underwater). I couldn't find any swimming functions or properties, too. Would 'jump' control state work for swimming upwards?

### PR DESCRIPTION
**Pull request by KaffinPX**
Closes #1882 
and adds a test about breathing

### CODE REVIEW

**Comment by KaffinPX** [comment]
will be fixed asap, its not ready bcs 1.8 metadata problems it works on 1.9 - 1.16.5

**Comment by KaffinPX** [comment]
Someone have an idea for fixing it?

**Comment by KaffinPX** [comment]
note: i wont fix it, if u want u can do

**Comment by rom1504** [comment]
tests are failing

**Comment by KaffinPX** [comment]
ik i need some help for making tests working

**Comment by KaffinPX** [comment]
> tests are failing

Working.

File: lib/plugins/breath.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module.exports = inject
+
+function inject (bot) {
+  bot._client.on('entity_metadata', (packet) => {
+    if (!bot?.entity?.id === packet?.entityId) return
+    if (packet?.metadata[1]?.key === 1) {
```
Comment (rom1504) [inline lib/plugins/breath.js:6]: Is 1 correct for all versions ?

Comment (KaffinPX) [reply lib/plugins/breath.js:6]: Yes

**Review by rom1504 — APPROVED** [review]

**Pull request merged**